snap/README.md

```diff
@@ -32,14 +32,6 @@ sudo snap connect edgexfoundry:edgex-secretstore-token edgex-device-usb-camera:e
 
 Please refer [here][secret-store-token] for further information.
 
-### Connect the camera interface
-The [`camera`](https://snapcraft.io/docs/camera-interface) interface is currently **NOT automatically connected** as it is pending auto-connection permissions from the store.
-
-To connect manually:
-```
-snap connect edgex-device-usb-camera:camera :camera
-```
-
```

Comment:
This isn't obsolete. The connection should still happen manually when the snap is built from source.